import-service/serverless.yml

```diff
@@ -40,10 +40,15 @@ functions:
       - http:
           path: /import
           method: get
+          cors: true
           request:
             parameters:
               querystrings:
                 name: true
+          authorizer: {
+            arn: 'arn:aws:lambda:${self:provider.region}:060881416814:function:authorization-service-dev-basicAuthorizer',
```

Comment:
you could try here like (not necessarily just a hint)
`arn: 'arn:aws:lambda:${self:provider.region}:#{AWS::AccountId}:function:authorization-service-dev-basicAuthorizer'`
          or
`arn: 'arn:aws:lambda:${self:provider.region}:${self:provider.accountId}:function:authorization-service-dev-basicAuthorizer'`
           maybe it will work